src/components/orgAuthTokenNote.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {useLocation} from '@reach/router';
+import {graphql, useStaticQuery} from 'gatsby';
+
+import {ExternalLink} from './externalLink';
+import {Note} from './note';
+import {SignedInCheck} from './signedInCheck';
+
+const siteMetaQuery = graphql`
+  query SignInNoteQuery {
+    site {
+      siteMetadata {
+        siteUrl
+      }
+    }
+  }
+`;
+
+export function OrgAuthTokenNote() {
+  const location = useLocation();
+  const data = useStaticQuery(siteMetaQuery);
+
+  const url = data.site.siteMetadata.siteUrl + location.pathname;
+
+  return (
+    <SignedInCheck isUserAuthenticated={false}>
+      <Note>
+        {' '}
+        You can{' '}
+        <ExternalLink href="https://sentry.io/settings/auth-tokens/" target="_blank">
+          manually create an Auth Token
+        </ExternalLink>{' '}
```

Comment:
doesn't the `ExternalLink` component already open the link in a new tab? asking because I am not familiar with the docs UI/components

Reply:
as far as I can tell, in docs it does not open in a new tab by default..?